src/containers/pages/FocusInvestigation/active/tests/__snapshots__/index.test.tsx.snap

```diff
@@ -479,11 +479,11 @@ Object {
     "canton": "Canton Tha Luang",
     "caseClassification": null,
     "caseNotificationDate": "2019-06-18",
-    "district": "",
+    "district": null,
     "focusArea": "TLv1_02",
     "plan_status": "complete",
     "plan_title": "A1-Tha Luang Village 1 Focus 01",
-    "province": "",
+    "province": null,
```

Comment:
@p-netm what is causing these changes here?

Reply:
I removed the util function that was cleaning up the jurisdiction data. `TransformValues`. it removed null values and replaced them with an empty string.I believe Its functionality is no longer required since the plans it was cleaning wont be used for display anymore.

Reply:
@p-netm thanks.  In the future please do remember to have such "why" information included in the commit message.  This one would have gone here https://github.com/onaio/reveal-frontend/pull/713/commits/8b831639390fa08973af83c1c737e36beff9371a

Reply:
https://chris.beams.io/posts/git-commit/

Reply:
ok